core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -1189,7 +1189,7 @@ class Partition(val topicPartition: TopicPartition,
    * @param minOneMessage whether to ensure that at least one complete message is returned
    * @param updateFetchState true if the Fetch should update replica state (only applies to follower fetches)
    * @return [[LogReadInfo]] containing the fetched records or the diverging epoch if present
-   * @throws NotLeaderOrFollowerException if this node is not the current leader and [[FetchParams.fetchOnlyLeader]]
+   * @throws NotLeaderOrFollowerException if this node is not the current leader and `FetchParams.fetchOnlyLeader`
```

Comment:
Are these changes related to this PR?

Reply:
TBH, I haven't found a way to link a Java method from scaladoc. Do you have any suggestion here?